Casks/lulu.rb

```diff
@@ -1,27 +1,17 @@
 cask "lulu" do
-  version "1.2.3"
-  sha256 "9c7bf4333af8dc7330614f2c9a389feac27334bc07f4b6039107b97eb476a3df"
+  version "2.0.0"
+  sha256 "0a7b7c2422aeca0b1cb45ae525c3882635fd603887bbb55e06bea81bd9ed69a8"
 
   # bitbucket.org/objective-see/ was verified as official when first introduced to the cask
-  url "https://bitbucket.org/objective-see/deploy/downloads/LuLu_#{version}.zip"
+  url "https://bitbucket.org/objective-see/deploy/downloads/LuLu_#{version}.dmg"
   appcast "https://objective-see.com/products/changelogs/LuLu.txt"
   name "LuLu"
   desc "Open-source firewall to block unknown outgoing connections"
   homepage "https://objective-see.com/products/lulu.html"
 
-  depends_on macos: ">= :sierra"
+  depends_on macos: ">= :big_sur"
```

Comment:
According to the [website](https://objective-see.com/products/lulu.html), the minimum supported OS is supported OS is macOS 10.15+ 😊 
```suggestion
  depends_on macos: ">= :catalina"
```